README.md

```diff
@@ -19,7 +19,7 @@ The code for Habitat itself is in the
 
 ## License
 
-Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
```

Comment:
@adamhjk you had a note about copyright years to me a bit ago. Including a range is unnecessary, yeah?